Sources/Web3Wallet/Web3Wallet.swift

```diff
@@ -35,10 +36,11 @@ public class Web3Wallet {
     /// - Parameters:
     ///   - metadata: App metadata
     ///   - signerFactory: Auth signers factory
-    static public func configure(metadata: AppMetadata, signerFactory: SignerFactory) {
+    static public func configure(metadata: AppMetadata, signerFactory: SignerFactory, echoHost: String = "echo.walletconnect.com", environment: APNSEnvironment) {
```

Comment:
should the echo host be an optional? not every project is using our PN solution.

Reply:
add echo config? that is optional and contains both?

Reply:
or it's good, just add a default environment?